client/src/components/Savings/ViewSavingsItem.js

```diff
@@ -0,0 +1,137 @@
+import React from 'react';
+import { Button } from "@mui/material"
+import DialogContent from '@mui/material/DialogContent';
+import DialogActions from '@mui/material/DialogActions';
+import IconButton from '@mui/material/IconButton';
+import CloseIcon from '@mui/icons-material/Close';
+import Typography from '@mui/material/Typography';
+import Dialog from '@mui/material/Dialog';
+import DialogTitle from '@mui/material/DialogTitle';
+
+import UpdateSavingItem from './UpdateSavingItem';
+import DeleteSavingItem from './DeleteSavingItem';
+
+/**
+ * Styling for dialog box
+ */
+const dialogTheme = {
+    '& .MuiDialogContent-root': {
+        padding: "20px",
+    },
+
+    '& .MuiDialogActions-root': {
+        padding: '15px',
+        'marginTop': '5px'
+    },
+    '& .MuiDialog-paper': {
+        "minWidth": "60%",
+    },
+
+
+}
+
+/**
+ * Styling for Dialog Headings 
+ */
+const DialogHeadings = {
+    fontWeight: 'bold',
+    "marginTop": '20px'
+}
+
+
+
+/**
+ * 
+ * @param {*} props item: the current row's item information
+ * @returns 
+ */
+const SavingsItemButton = (props) => {
+    const [open, setOpen] = React.useState(false);
+
+    const handleClickOpen = () => {
+        setOpen(true);
+    };
+    const handleClose = () => {
+        setOpen(false);
+    };
+
+    return (
+        <div>
+            <Button onClick={handleClickOpen}>
+                More Info
+            </Button>
+
+            <Dialog
+                onClose={handleClose}
+                aria-labelledby="customized-dialog-title"
+                open={open}
+                sx={dialogTheme}
+            >
+                <DialogTitle sx={{ m: 0, p: 2 }}>
+                    View Saving
+                    <IconButton
+                        aria-label="close"
+                        onClick={handleClose}
+                        sx={{
+                            position: 'absolute',
+                            right: 17,
+                            top: 13,
+                        }}
+                    >
+                        <CloseIcon />
+                    </IconButton>
+                </DialogTitle>
+
+                <DialogContent dividers>
+                    <Typography sx={DialogHeadings}>
+                        Merchant Name:
```

Comment:
Probably should update the heading name to source as well

Reply:
thanks! i missed that